File: Java/백준/Bronze/11121. Communication Channels/Communication Channels.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int q = in.nextInt();
        for (int i =0; i<q; i++){
            String a = in.next();
            String b = in.next();
            if(a.equals(b)){
                System.out.println("OK");
            }
            else{
                System.out.println("ERROR");
            }
        }
    }
}

File: C#/백준/Bronze/11121. Communication Channels/Communication Channels.cs
using System;

namespace q{
    class w{
        static void Main(){
            int a = int.Parse(Console.ReadLine());
            for (int i=0; i<a; i++){
                String[] aa = Console.ReadLine().Split();
                if(aa[0].Equals(aa[1])){
                    Console.WriteLine("OK");
                }
                else{
                    Console.WriteLine("ERROR");
                }
            }
        }
    }
}